memory/memtester.py.data/memtester.yaml

```diff
@@ -1,7 +1,8 @@
 test_memster:
     memtest_opt:
         runs: 1
-        memory: '1G'
+        # NOTE: Specify in MB
+        memory: 1024
         # NOTE: Provide page boundary address
-        physaddr: 0x100000
-        device: '/dev/mem'
+        # physaddr: 0x100000
```

Comment:
may be left these as blank # denote that it is not supported

Reply:
and same need to handle in test script

Reply:
these are supported values, but depends on page boundary on each system, hence commented for reference.

Reply:
ok I understand but commenting express like  it is not supported that's why requested pass as blank or None which you feel should be better and handle in the script if it None or blank  skip that test

Reply:
OK